routes/userRegistration.js

```diff
@@ -0,0 +1,12 @@
+var express = require('express');
+var router = express.Router();
+
+let userRegistrationController = require("../controllers/userResgistration");
+
+router.get("/users", userRegistrationController.getAllUsers);
+router.post("/create", userRegistrationController.createUser);
+router.get("/get/:username", userRegistrationController.userByusername);
+router.put("/edit/:username", userRegistrationController.update);
+router.delete("/delete/:username", userRegistrationController.deleteUser);
```

Comment:
```suggestion
router.get("/", userRegistrationController.getAllUsers);
router.post("/", userRegistrationController.createUser);
router.get("/:username", userRegistrationController.userByusername);
router.put("/:username", userRegistrationController.update);
router.delete("/:username", userRegistrationController.deleteUser);
```
These can be simplified as the Method defines what action should be taken